pyproject.toml

```diff
@@ -1,6 +1,91 @@
+[tool.poetry]
+name = "phc-sdk-py"
+version = "0.32.0"
+description = "Python SDK for the LifeOmic platform"
+authors = ["LifeOmic <development@lifeomic.com>"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/lifeomic/phc-sdk-py"
+documentation = "https://lifeomic.github.io/phc-sdk-py/"
+packages = [{ include = "phc" }]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: MIT License",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Healthcare Industry",
+  "Natural Language :: English",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Topic :: Utilities",
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+aiohttp = ">3.5.3,<4.0.0"
+backoff = ">=1.10.0,<2.0.0"
+nest-asyncio = ">=1.5.6,<2.0.0"
+pyjwt = ">=2.4.0,<3.0.0"
+pandas = ">=1.3.5,<2.0.0"
+funcy = ">=2.0,<3.0.0"
+lenses = ">=1.1.0,<2.0.0"
+toolz = ">=0.12.0,<1.0.0"
+pydantic = ">=1.10.2,<2.0.0"
+python-pmap = ">=2.0.0,<3.0.0"
+fastparquet = ">=0.8.1,<1.0.0"
```

Comment:
For future reference: Poetry supports semver, so these dependency requirements could be simplified to e.g. on this line `^0.8.1`, which is equivalent to `>=0.8.1,<1.0.0`.